ownCloud/Client/ClientItemCell.swift

```diff
@@ -90,6 +101,34 @@ class ClientItemCell: ThemeTableViewCell {
 			self.accessoryType = .none
 		}
 
+		if item.thumbnailAvailability != .none {
+			let displayThumbnail = { (thumbnail: OCItemThumbnail?) in
+				thumbnail?.requestImage(for: thumbnailSize, scale: 0, withCompletionHandler: { (thumbnail, error, _, image) in
+					if error == nil {
+						DispatchQueue.main.async {
+							if image != nil {
+								if self.item?.versionIdentifier == thumbnail?.versionIdentifier {
+									self.iconView.image = image
+								}
+							}
+						}
+					}
+				})
+			}
+
+			if let thumbnail = item.thumbnail {
+				_ = displayThumbnail(thumbnail)
```

Comment:
I think the displayThumbnail closure is taking the unused `thumbnail?.requestImage(...)` Bool return as it's own return parameter and this is why it's saying to you `Result of call is unused, but produces 'Bool?'` when you later  call  `displayThumbnail(thumbnail)` if you don't assign it's return value to some variable ( in this case to a `_`)

You can get rid of this `_` assignment if you code it here:
```Swift
_ = thumbnail?.requestImage(for: thumbnailSize, scale: 0, withCompletionHandler: { (thumbnail, error, _, image)
```
then you can call the `displayThumbnail(thumbnail)` without any assignment: 
```Swift
displayThumbnail(thumbnail)
```

Reply:
Thanks. Made that change as well.